### DISCUSSION TITLE
Glop solver giving different optimal values in Linux and windows environment but optimal solution is same with slight changes after decimal point

### DISCUSSION THREAD

**Discussion by mygit-prash** [google/or-tools#3232] [Linear Solver questions] [answered]
Glop solver giving different optimal values in Linux and windows environment but optimal solution is same with slight changes after decimal point.

How can we achieve same results in both Linux and windows environment while using Glop solver for linear optimization problem. 

Do we need to add any tolerance value to get only one type of solution irrespective of environment and change in order of constraints.

Even changing the order of soft and hard constraints we get different outcome. 

Any solutions for these will be appreciated.

**Comment by Mizux** [comment]
same hardware ?
does this decimal number are just additive noise ?

**Comment by lperron** [comment, marked as answer]
Already answered
 What is on S.O. ?

**Reply by lperron** [reply]
actually in the issue part.

As I said, numerical operations are actually compiler specific.
